waku.md

```diff
@@ -370,6 +371,21 @@ In order to receive historic messages from a mailserver, a node MUST trust the s
 
 Received envelopes MUST be passed through the Waku envelopes pipelines so that they are picked up by registered filters and passed to subscribers.
 
+### Accounting for resources (experimental)
+
+Nodes MAY implement accounting, keeping track of resource usage. It is heavily inspired by Swarm's SWAP protocol, and works by doing pairwise accounting for resources.
```

Comment:
Link to swap?

Reply:
```suggestion
Nodes MAY implement accounting, keeping track of resource usage. It is heavily inspired by Swarm's [SWAP protocol](https://www.bokconsulting.com.au/wp-content/uploads/2016/09/tron-fischer-sw3.pdf), and works by doing pairwise accounting for resources.
```

this?

Reply:
no canonical link yet IIRC, their specs are in PR